packages/scrollable-image/ui/src/components/embedded-code-modal.js

```diff
@@ -1,9 +1,8 @@
+import orangutan from '@twreporter/orangutan'
```

Comment:
Wouldn't `import orangutan from '@twreporter/orangutan'` cause circular dependencies?

In current dependency tree,
`@twreporter/orangutan`  installs `@twreporter/scrollable-image`
`@twreporter/scrollable-image` installs `@twreporter/orangutan`